osu.Game.Tests/Visual/SongSelectV2/TestSceneSongSelect.cs

```diff
@@ -64,6 +104,12 @@ public override void SetUpSteps()
 
             AddStep("load screen", () => Stack.Push(new Screens.SelectV2.SongSelectV2()));
             AddUntilStep("wait for load", () => Stack.CurrentScreen is Screens.SelectV2.SongSelectV2 songSelect && songSelect.IsLoaded);
+
+            AddStep("download bundled beatmaps", () =>
```

Comment:
Surely this wasn't supposed to stay around. Unless you're going to mark this as a non-headless test or something..

Reply:
The intention was to make this non-headless, but I've removed it as per discussion.